### ISSUE TITLE
Color contrast improvements for Summer and Night

### ISSUE THREAD

**Issue by aviannamiller** [omeka/theme-seasons#40]
On Summer:
- In search bar, Advanced and Submit Search buttons use #ffffff for icons against a #8ECAC8 background (only a 1.8:1 contrast)
  - Same goes for buttons on advanced item search (Add a Field, Search for Items)

On Night:
- In search bar, Advanced and Submit Search buttons use #666666 for icons against a #252525 background (only a 2.7:1 contrast)
  - Same goes for buttons on advanced item search (Add a Field, Search for Items)
- #6d6d6d `.nav-item li` are only at a 2.8:1 contrast on the #BDBDBD nav bar
- #FFE000 `.sub-nav li` on the #666 sub nav are only at a 4.4:1 contrast

**Comment by aviannamiller** [comment]
These all look great, thanks! One additional request relevant to both stylesheets:
- The Advanced Search button on the search bar has a hover state that styles the icon at that #ffffff. Can that either be changed or removed?

And another observation pertinent to all stylesheets:
- It looks like Skip Nav links are informed by the general link styles, most of which have recently been altered to meet contrast requirements for LC project links in https://github.com/omeka/plugin-LocalContexts/issues/3. However, the Skip Nav links are now failing minimum contrast needs against the #404040 `admin-bar`. These likely each need their own style in order to insure they are not impacted by the link style change, or a generally acceptable color across themes (e.g. #ffffff) (although I'm not totally sure if they would have had proper contrast before the changes). Additionally, they receive the `a:visited` style, which likely does not make sense for the utility of a skip link (and for Night, causes an issue in which its basic link style passes contrast requirements for the skip link but fails when the visited link style applies instead).

**Comment by kimisgold** [comment]
Okay, both these issues should be addressed and are ready for a look.

**Comment by aviannamiller** [comment]
Fixes to Spring look great. For Autumn, that `#top-nav` focus style is still applying to `.nav-item li` (at about a 2.1:1 contrast). Meanwhile, those li are also receiving `nav.top` link styles, which are #BD592A on the #DFD2B5 background (3:1):

<img width="683" height="417" alt="Detail of nav menu, displaying three different states with three distinct styles on list items: plain link/visited, focus, and hover/open. The plain link/visited and focus states have poor contrast." src="https://github.com/user-attachments/assets/c773c357-6cff-4390-a9d1-f2087c243100" />

**Comment by aviannamiller** [comment]
Oh sorry! I see you just put in changes for the subnav.

**Comment by aviannamiller** [comment]
Thank you! No remaining issues; this is good to close.
